locale/en.po

```diff
@@ -6099,6 +6099,18 @@ msgstr "UV protection too low, needs to be {0}% higher"
 msgid "TOLERANCES_UNSUITABLE_DEBUFFS"
 msgstr "Not adapted to current environmental conditions. Osmoregulation cost {0}%, bioprocess speed {1}%, health {2}%"
 
+msgid "TOLERANCE_DETAIL_TEXT"
+msgstr "[b]Species[/b]\n"
```

Comment:
I think you've accidentally reset the text?

Also testing in game seems to confirm that this results in quite non-sensical display:

<img width="700" height="703" alt="kuva" src="https://github.com/user-attachments/assets/f4f7d675-3fbe-4927-818c-e2c1e919c312" />

And I think in the code you should at least convert the oxygen and UV resistances to percentages for easier display to the player.